### ISSUE TITLE
Issues running sparqlalgebrajs

### ISSUE THREAD

**Issue by jitsedesmet** [joachimvh/SPARQLAlgebra.js#81]
I get an error when running sparqlalgebrajs. I installed sparqlalgebrajs by running `sudo npm install -g sparqlalgebrajs`. Whenever I run the commend I get an `Error: Cannot find module '../test/util'`
Stack trace:

```
sparqlalgebrajs -q "SELECT ?x ?y ?z WHERE { ?x ?y ?z }"
node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module '../test/util'
Require stack:
- /usr/lib/node_modules/sparqlalgebrajs/bin/sparqlalgebrajs.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/usr/lib/node_modules/sparqlalgebrajs/bin/sparqlalgebrajs.js:7:16)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/lib/node_modules/sparqlalgebrajs/bin/sparqlalgebrajs.js' ]
}
```

**Comment by jitsedesmet** [comment]
I didn't try that. I cloned the repo and executed the command like that.

**Comment by joachimvh** [comment]
Fixed in b8ee584eb8234e932711cc911cbd83f10725b70c
